Makefile

```diff
@@ -43,5 +52,7 @@ dist/build: $(SOURCE_FILES)
 	touch dist/build
 
 inspect: build
-	node --inspect dist/app.js
+	node --inspect dist/src/app.js
```

Comment:
I hope we can revert this change. I ran into the same issue and it was because `knexfile.ts` is in the root.

Can we move `knexfile.ts` into `src/` ? The `npx knex` command has a `--knexfile` option to point to the right place.

I also did this here:

https://github.com/badgateway/tt/blob/main/package.json#L14

Reply:
Anytime that we add another entry in tsconfig.js that is _outside_ of the `src/` directory it will distrupt how the `dist/` folder willl look. So if we want to revert the `dist/src/app.js` changes we would have to be okay with both the `knexfile.ts` and `migrations/` residing in `src/`, is that preferred?

Reply:
I think so, yes!